internal/web/context.go

```diff
@@ -0,0 +1,22 @@
+package web
+
+import (
+	"github.com/labstack/echo/v4"
+	"go.uber.org/zap"
+)
+
+type AppContext struct {
+	echo.Context
+	AppLogger *zap.Logger
+}
+
+func CreateAppContext(
```

Comment:
The logger is initialized directly in the AppContext without any validation or checks. Consider checking if the logger is nil before using it to prevent dereferencing a nil pointer which might cause panic.